scopus-import/src/main/java/no/sikt/nva/scopus/conversion/PublicationChannelConnection.java

```diff
@@ -53,16 +58,25 @@ private static PublicationChannelResponse toPublicationChannelResponse(String re
     }
 
     private Optional<URI> fetchPublicationChannelId(Stream<URI> uriStream) {
-        return uriStream.map(uri -> uriRetriever.getRawContent(uri, CONTENT_TYPE))
+        return uriStream.map(uri -> uriRetriever.fetchResponse(uri, CONTENT_TYPE))
                    .filter(Optional::isPresent)
                    .map(Optional::get)
+                   .map(this::handleResponse)
                    .map(PublicationChannelConnection::toPublicationChannelResponse)
                    .filter(Objects::nonNull)
                    .filter(PublicationChannelConnection::containsSingleResult)
                    .map(PublicationChannelConnection::getId)
                    .findFirst();
     }
 
+    private String handleResponse(HttpResponse<String> response) {
+        if (HTTP_OK != response.statusCode()) {
+            logger.error("Publication channels API responded with {} when searching for {}", response.statusCode(),
+                         response.request().uri());
```

Comment:
I hope that the escaping here should is very robust before we request

Reply:
UriWrapper is handling escaping yes.